tool/dev.dart

```diff
@@ -24,9 +23,8 @@ Future<Null> main(List<String> args) async {
     ..strong = true
     ..fatalLints = true
     ..entryPoints = ['benchmark/', 'lib/', 'test/', 'example/'];
-  
-  config.format
-    ..paths = ['benchmark/', 'lib/', 'test/', 'example/'];
+
+  config.format..paths = ['benchmark/', 'lib/', 'test/', 'example/'];
 
   await dev(args);
-}
```

Comment:
Since this ddev config is simple, we should be able to upgrade it to ddev v3 without much effort